### ISSUE TITLE
Can't get prev/next arrows to work

### ISSUE THREAD

**Issue by NicholasJohn16** [imgix/luminous#74]
I wanted to see with the arrow navigation looked like in Luminous, so I download a zip of the repo real quick, edited the index.html file and changed the js to the following:

```new LuminousGallery(
document.querySelectorAll('.gallery-demo'), 
{
  arrowNavigation: true
}, 
{
  caption: function (trigger) {
    return trigger.querySelector('img').getAttribute('alt');
  }
});
```

But the arrow navigation doesn't seem to work. No new icons appear and the arrow keys don't do anything. Am I missing something? Thank you for your help and great library.

**Comment by frederickfogerty** [comment]
Hey there @NicholasJohn16, sorry to hear that you've had issue with Luminous! This is on my list of issues to look at for the next two weeks. I'll keep this issue updated with my progress.

**Comment by hellorich** [comment]
Hi @frederickfogerty - I've had what I think may be similar issues with the Gallery functionality as @NicholasJohn16  - I'm not entirely sure its working as intended, either locally in my project (standard Luminous lightbox seems fine) nor in codepens I've based on the Lightbox example, modified using Gallery code in the readme.

Here's one of my pens: https://codepen.io/anon/pen/RegwWp

It seems the first image loads in a lightbox, but any others simply load the linked image.

**Comment by SiamKreative** [comment]
Same issue here

**Comment by AlexisWalravens** [comment]
Same issue here, was there any progress on this ?

**Comment by frederickfogerty** [comment]
Hi all. This was a regression from a previous release and is fixed in the PR #121. If you would like to checkout that PR and test that it fixes the problem, that would be much appreciated. Otherwise, sit tight and a fix should be released soon!
